src/filesystem_helper.cpp

```diff
@@ -341,6 +343,38 @@ path create_temp_directory(const std::string & base_name, const path & parent_pa
   return final_path;
 }
 
+std::filesystem::path create_temporary_directory(
```

Comment:
#include `<filesystem>`

Reply:
I've added it to the relevant header file. Therefore no need to add it to the cpp file.